src/arrayMethodSort.js

```diff
@@ -4,8 +4,24 @@
  * Implement method Sort
  */
 function applyCustomSort() {
-  [].__proto__.sort2 = function(compareFunction) {
-    // write code here
+  [].__proto__.sort2 = function (
+    compareFunction = (a, b) => {
+      if (a === b) {
```

Comment:
Using strict equality (===) to compare the elements might not be necessary since you're converting both elements to strings immediately after. You could simply use `String(a) === String(b)`.